R/ducktbl.R

```diff
@@ -7,37 +7,37 @@
 #' For such objects,
 #' dplyr verbs such as [mutate()], [select()] or [filter()]  will use DuckDB.
 #'
-#' `duckdb_tibble()` works like [tibble()], returning an "eager" duckplyr data frame by default.
-#' See the "Eager and lazy" section below.
+#' `duckdb_tibble()` works like [tibble()], returning an "untethered" duckplyr data frame by default.
+#' See the "Tethering" section below.
 #'
-#' @section Eager and lazy:
+#' @section Tethering:
 #' Data frames backed by duckplyr behave as regular data frames in almost all respects.
 #' In particular, direct column access like `df$x`,
 #' or retrieving the number of rows with [nrow()], works identically.
 #' For a duckplyr frame that is the result of a dplyr operation,
 #' accessing column data or retrieving the number of rows will trigger a computation
 #' that is carried out by DuckDB.
 #'
```

Comment:
For the next paragraph, lead with the problem: materializing is not always affordable, then introduce 'tethering'?